providers/aws/docdb.go

```diff
@@ -0,0 +1,141 @@
+// Copyright 2019 The Terraformer Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package aws
+
+import (
+	"context"
+	"log"
+
+	"github.com/GoogleCloudPlatform/terraformer/terraformutils"
+	"github.com/aws/aws-sdk-go-v2/service/docdb"
+)
+
+var docDBAllowEmptyValues = []string{"tags."}
+
+type DocDBGenerator struct {
+	AWSService
+}
+
+func (g *DocDBGenerator) InitResources() error {
+	config, e := g.generateConfig()
+	if e != nil {
+		return e
+	}
+	svc := docdb.NewFromConfig(config)
+
+	if err := g.getClusters(svc); err != nil {
+		log.Println(err)
+	}
+
+	if err := g.getSubnetGroups(svc); err != nil {
+		log.Println(err)
+	}
+
+	if err := g.getParameterGroups(svc); err != nil {
+		log.Println(err)
+	}
+
+	return nil
+}
+
+func (g *DocDBGenerator) getClusters(svc *docdb.Client) error {
+	clusterPaginator := docdb.NewDescribeDBClustersPaginator(svc, &docdb.DescribeDBClustersInput{})
```

Comment:
This PR have a bug.
We need to add filter when describeDBCluster
https://docs.aws.amazon.com/documentdb/latest/developerguide/API_DescribeDBClusters.html